roles/common/tasks/install-Debian.yml

```diff
@@ -1,13 +1,14 @@
 ---
 
 - name: Update package cache (Debian)
-  apt: update_cache=yes cache_valid_time=86400
+  apt: 
+    update_cache: yes 
+    cache_valid_time: 86400
   become: yes
 
 - name: Ensure common packages are installed (Debian)
-  apt: pkg={{ item }} state=present install_recommends=no
-  with_items:
-    - aptitude
-    - ca-certificates
-    - vim-nox
+  apt: 
+    pkg: ['aptitude', 'ca-certificates', 'vim-nox']
```

Comment:
Could you please split this into multiple lines, i.e.:
```
pkg:
  - aptitude
  - ca-certificates
  - vim-nox
```

Reply:
Done!